### DISCUSSION TITLE
How can I build this?

### DISCUSSION THREAD

**Discussion by doragasu** [jonathanrjpereira/DWM1000-Real-Time-Localization-System#3] [Q&A] [answered]
Hi, thanks for the project, it is very interesting.

What hardware platform did you use for the project, maybe the nodemcu-BU01? And how do you build it? Which libraries/firmware does it need and where can I find them?

Thanks!

**Comment by jonathanrjpereira** [comment, marked as answer]
- This project is a basic RTLS system that can track an object in 2D space (Can be easily modified to work in 3D space).
- You will need to buy four [Decawave DWM1001 Dev Boards](https://www.decawave.com/product/dwm1001-development-board/). 3 of these will be configured as fixed Anchors and the fourth board will be configured as a moveable (trackable) Tag.
- Once you buy the boards you will need to install the Segger Embedded Studio IDE and flash PANS2.0  on all four boards. The [DWM1001 Firmware User Guide](https://www.decawave.com/wp-content/uploads/2019/01/DWM1001-Firmware-User-Guide-2.1.pdf) will show you how to install the Software Toolchain and also how to flash PANS2.0 onto the dev boards.
- The code in this repository (uses the PANS2.0 library) will help you to configure the boards as anchors/tag, set the anchor positions, retrieve the real-time tag location, and also visualize the tag/anchors in a 2D space.

**Reply by doragasu** [reply]
Thank you very much for the detailed explanation, it is clear for me now!
